views/js/front.js

```diff
@@ -113,6 +116,5 @@ function validateBlikCode(blik_code_value)
     } else {
         $payment_button.prop('disabled', true);
         $paynow_blik_payment_button.prop('disabled', true);
-        $('#paynow_blik_code').focus();
```

Comment:
nie usuwałbym całkowicie tego focusa, tylko może akcje robił kiedy się pojawi formularz

Reply:
No ale, to tak działało do tej pory, tj. następowało scrollowanie kiedy formularz stawał się aktywny (po odsłonięciu) i tego dotyczyło zgłoszenie #150.  Jeśli mamy one-page-checkout to staje się to uciążliwe, bo strona skacze do dołu kiedy formularz się odsłania, zatem robienie focusa raz kiedy się pojawi nie rozwiąże tego problemu - wręcz jest to przyczyna problemu.

Zamiast wywalenia tego na stałe, możemy ewentualnie dodać settings we wtyczce, który wyłączy 
focus(scrollowanie) i każdy będzie mógł w konfiguracji wybrać opcję, czy ma ma się ten focus/scroll pojawić czy nie.

Reply:
może trzeba sprawdzić widoczność któregoś elementu. W sklepach poza The Checkout będzie to działało po usunięciu słabo, bo raczej  strona przewinie się do błędu na górze strony.

Reply:
warunkowo wpuszczam, ale ten temat UX jest do poprawienie